### PR TITLE
State non mandatory fields as optional

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
This PR fixes the import application-version command to allow optional fields to be non-filled in 

### How to prepare for test
- [x] ssh to relevant server (depending on type of change)
- [x] Use stage: `us`
- [x] paxa the environment: `paxa`
- [x] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [x] do import application-version excel with the optional fields not filled in

### Expected test outcome
- [x] check that no error occurs
- [x] Take a screenshot and attach or copy/paste the output.

## Review
- [x] code approved by MR
- [x] tests executed by PG
- [x] "Merge and deploy" approved by PG
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [x] Document in changelog
- [ ] Deploy this branch

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
@moonso do you want to review this as it is an pydantic schema change?

File: cg/store/api/models.py [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
-    standard: int = Field(alias="Standard")
-    priority: int = Field(alias="Priority")
-    express: int = Field(alias="Express")
-    research: int = Field(alias="Research")
-    percent_kth: int = Field(alias="Percent kth")
+    standard: Optional[int] = Field(alias="Standard")
+    priority: Optional[int] = Field(alias="Priority")
+    express: Optional[int] = Field(alias="Express")
+    research: Optional[int] = Field(alias="Research")
+    percent_kth: Optional[int] = Field(alias="Percent kth")
```
Comment (Mropat) [inline cg/store/api/models.py:17]:
```suggestion
```

File: cg/store/api/models.py [hunk 2]
```diff
@@ -8,13 +8,13 @@
 
 class ApplicationVersionSchema(BaseModel):
     app_tag: str = Field(alias="App tag")
-    version: int = Field(alias="Version")
+    version: Optional[int] = Field(alias="Version")
```
Comment (Mropat) [inline cg/store/api/models.py:11]:
```suggestion
    version: int = Field(alias="Version")
```

**Review by Mropat — APPROVED** [review]

File: cg/store/api/models.py (hunk 2 above)
Comment (patrikgrenfeldt) [reply cg/store/api/models.py]: fixed with a parse model

**Comment by patrikgrenfeldt** [comment]
Tested locally due to misbehaving update on Hasta:
image
🆗

**Comment by patrikgrenfeldt** [comment]
Deployed on Hasta:
image

**Pull request merged**